extensions/wikia/TemplateDraft/controllers/TemplateDraftController.class.php

```diff
@@ -35,4 +35,23 @@ public function createDraftContent( Title $title, $content, Array $flags ) {
 
 		return $newContent;
 	}
+
+	public function markTemplateAsNotInfobox() {
+		$requestParams = $this->getRequest()->getParams();
+
+		if ( empty( $requestParams['pageId'] )
+			|| !is_numeric( $requestParams['pageId'] )
```

Comment:
Instead of using `getParams` when you only have one parameter right now, it might be better to just use `$pageId = this->getRequest()->getInt( 'pageId' );` and checking it isn't `0`.